### PR TITLE
All packages upgraded to latest

### PR DESCRIPTION
**Pull request by garvit-rajput**
### What does this PR do?

This PR upgrades all packages to latest maintaining compatibility with angular 9

### Changes

codelyzer changes
https://github.com/mgechev/codelyzer/blob/master/CHANGELOG.md#500-2019-03-27

prettier changes 
https://prettier.io/blog/2020/03/21/2.0.0.html

### What other information should the reviewer be aware of when looking at this code?

These changes have not been tested as storybook is breaking.

### Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -49,7 +49,6 @@
     "no-trailing-whitespace": true,
     "no-unnecessary-initializer": true,
     "no-unused-expression": true,
-    "no-use-before-declare": true,
```
Comment (mderazon) [inline tslint.json:52]: @garvit-rajput why did we remove this rule ? Haven't seen it in the deprecation list

File: ui/src/components/autocomplete/autocomplete.spec.ts [hunk 2]
```diff
@@ -153,7 +153,7 @@ class NgIfAutocomplete {
 
   constructor() {
     this.filteredOptions = this.optionCtrl.valueChanges.pipe(
-      startWith(null),
```
Comment (mderazon) [inline ui/src/components/autocomplete/autocomplete.spec.ts:156]: What was the change here ?

Comment (garvit-rajput) [reply ui/src/components/autocomplete/autocomplete.spec.ts:156]: https://github.com/ReactiveX/rxjs/issues/4772

File: tslint.json (hunk 1 above)
Comment (garvit-rajput) [reply tslint.json:52]: It was showing deprecation warnings while running lint checks
https://github.com/palantir/tslint/pull/3919

It says compiler should be handling this.

Also found this https://github.com/palantir/tslint/issues/4100

**Review by mderazon — APPROVED** [review]

**Pull request merged**